ts/WoltLabSuite/Core/Acp/Ui/DataImport/Manager.ts

```diff
@@ -0,0 +1,135 @@
+/**
+ * Provides the program logic for the data import function.
+ *
+ * @author  Marcel Werk
+ * @copyright  2001-2021 WoltLab GmbH
+ * @license  GNU Lesser General Public License <http://opensource.org/licenses/lgpl-license.php>
+ * @module  WoltLabSuite/Core/Acp/Ui/DataImport/Manager
+ * @woltlabExcludeBundle all
+ */
+import * as Ajax from "../../../Ajax";
+import * as Core from "../../../Core";
+import * as Language from "../../../Language";
+import { AjaxCallbackObject, AjaxCallbackSetup, ResponseData } from "../../../Ajax/Data";
+import { AjaxResponse } from "../../../Controller/Clipboard/Data";
+import { DialogCallbackSetup } from "../../../Ui/Dialog/Data";
+import DomUtil from "../../../Dom/Util";
+import UiDialog from "../../../Ui/Dialog";
+
+export class AcpUiDataImportManager implements AjaxCallbackObject {
```

Comment:
Generally you would avoid `AjaxCallbackObject` in new code and instead use the promise-based `dboAction`.

Reply:
`dboAction` can only call `AjaxProxyAction` but I have to call `WorkerProxyAction`.

Reply:
Ah, indeed. I misread the URL in `_ajaxSetup()`.